### DISCUSSION TITLE
Allowing div tags

### DISCUSSION THREAD

**Discussion by blue4euro4** [ueberdosis/tiptap#2612] [General Questions]
![image](https://user-images.githubusercontent.com/52677475/157918996-c4405d75-6c36-436a-8d82-c6756a4c549d.png)

The above is the code i'm using to allow div tags and it works, however whenever I click enter it closes any div tags open and then starts creating paragraphs. I want it to only allow div tags in the HTML but not mess with it in the editor itself. I want the editor to only create paragraph tags when I press enter. Is it possible to have it like this and if so where am I going wrong?

Thanks in advance!

**Comment by philippkuehn** [comment]
You probably want to set a higher `priority` for that div node: https://tiptap.dev/guide/custom-extensions#priority
